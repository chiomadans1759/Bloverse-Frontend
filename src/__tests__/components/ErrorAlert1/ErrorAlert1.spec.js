import { mount } from '@vue/test-utils';
import ErrorAlert1 from '@/components/ErrorAlert1/ErrorAlert1.vue';

describe('ErrorAlert1 Component', () => {
  it('should render correctly', () => {
    const wrapper = mount(ErrorAlert1);

    expect(wrapper.element).toMatchSnapshot();
  });
  it('should render title', () => {
    const wrapper = mount(ErrorAlert1, {
      propsData: {
        heading: 'alert-heading'
      }
    })
    expect(wrapper.props().heading).toBe('alert-heading');
  })
});
