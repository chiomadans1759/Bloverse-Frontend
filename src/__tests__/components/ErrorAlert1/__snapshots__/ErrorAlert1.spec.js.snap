// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorAlert1 Component should render correctly 1`] = `
<main
  id="error-alert"
>
  <div
    class="alert alert-danger error"
    role="alert"
  >
    <!---->
     
    <p
      class="text"
    />
  </div>
</main>
`;
